### PR TITLE
code review

### PR DESCRIPTION
**Pull request by Obito-git**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+backend/target
```
Comment (Obito-git) [inline .gitignore:1]: target should not be in the project

File: backend/src/main/java/samoyla/todo_list/TodoListApplication.java [hunk 2]
```diff
@@ -1,13 +1,10 @@
-package samoyla.TodoList;
+package samoyla.todo_list;
```
Comment (Obito-git) [inline backend/src/main/java/samoyla/todo_list/TodoListApplication.java:1]: package name is alwals in lowercase

File: backend/src/main/java/samoyla/todo_list/TodoListApplication.java [hunk 3]
```diff
@@ -1,13 +1,10 @@
...
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
-@ComponentScan(basePackages = {"samoyla.TodoList", "samoyla.TodoList.mapper"})
```
Comment (Obito-git) [inline backend/src/main/java/samoyla/todo_list/TodoListApplication.java:8]: spring scans it by itself

File: backend/src/main/java/samoyla/todo_list/dto/TaskDto.java [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
-public class TaskDTO {
 
+public class TaskDto {
     private Long id;
     private String content;
     private boolean completed;
     private String type;
 
-    public TaskDTO() {
+    public TaskDto() {
```
Comment (Obito-git) [inline backend/src/main/java/samoyla/todo_list/dto/TaskDto.java:10]: Minor change, but better to have Dto to respect the naming convention

File: backend/src/main/java/samoyla/todo_list/dto/TaskDto.java [hunk 5]
```diff
@@ -41,5 +41,18 @@ public String getType() {
     public void setType(String type) {
         this.type = type;
     }
-    
+
+}
+
+
```
Comment (Obito-git) [inline backend/src/main/java/samoyla/todo_list/dto/TaskDto.java:47]: Great example why lombok exists

File: backend/src/main/java/samoyla/TodoList/config/MappingConfig.java
Comment (Obito-git) [inline backend/src/main/java/samoyla/TodoList/config/MappingConfig.java:1]: Idk where did u find this approach, but it was strange

File: backend/src/main/java/samoyla/todo_list/TodoListApplication.java (hunk 2 above)
Comment (samoyla) [reply backend/src/main/java/samoyla/todo_list/TodoListApplication.java:1]: ok chief

File: backend/src/main/java/samoyla/todo_list/TodoListApplication.java (hunk 3 above)
Comment (samoyla) [reply backend/src/main/java/samoyla/todo_list/TodoListApplication.java:8]: got it !

File: backend/src/main/java/samoyla/todo_list/dto/TaskDto.java (hunk 4 above)
Comment (samoyla) [reply backend/src/main/java/samoyla/todo_list/dto/TaskDto.java:10]: got it Dto !

File: .gitignore (hunk 1 above)
Comment (samoyla) [reply .gitignore:1]: i think it creates each time i use mvn clean install or smth, so the directory is created as part of the build process

File: backend/src/main/java/samoyla/todo_list/dto/TaskDto.java (hunk 5 above)
Comment (samoyla) [reply backend/src/main/java/samoyla/todo_list/dto/TaskDto.java:47]: i still didn't read about lombok

**Review by samoyla — APPROVED** [review]

File: backend/src/main/java/samoyla/TodoList/config/MappingConfig.java
Comment (samoyla) [reply backend/src/main/java/samoyla/TodoList/config/MappingConfig.java:1]: don't even ask :)

**Pull request merged**